docs/manual/job-options.md

```diff
@@ -446,31 +446,33 @@ The path must indicate a stored `password` entry in the storage facility.
 
 ![Storage Path for Secure Option](~@assets/img/jobs-options-secure-storage-path.png)
 
-## Remote option values
+## Option Model Provider
 
-A model of option values can be retrieved from an external source
-called an _option model provider_.
-When the `valuesUrl` is specified for an Option, then the model of
-allowed values is retrieved from the specified URL.
+### Remote URL Job Options
 
-This is useful in a couple of scenarios when Rundeck is used to
-coordinate process that depend on other systems:
+![Remote URL Job Options](@assets/img/remote-url-job-options.png)<br>
 
-- Deploying packages or artifacts produced by a build or CI server, e.g. Jenkins.
+Remote URL Job Options provides a method for retrieving a list of Job Option values from a remote service or file.
+
+Option values can be retrieved from an external source called an _option model provider_.
+
+This is useful in a couple of scenarios when Process Automation is used to
+coordinate processes that depend on other systems:
+
+- Deploying packages or artifacts produced by a build or CI server, such as Jenkins:
   - A list of recent Jenkins build artifacts can be imported as Options data, so that a User can pick an appropriate package name to deploy from a list.
 - Selecting from a set of available environments, defined in a CMDB
 - Any situation in which input variables for your Jobs must be selected from some set of values produced by a different system.
 
-## Option model provider
+When defining the Job in YAML, XML, or JSON the `valuesUrl` is specified for an Option, then the model of
+allowed values is retrieved from the specified URL:
 
-The Option model provider is a mechanism to allow the Options defined for a Job to have some of the possible input values provided by a remote service or database.
-
-Option model providers are configured on a per-Option basis (where a Job may have zero or more Options).
+    <option valuesUrl="http://site.example.com/values.json" ... />
 
 ### Requirements
 
-1. Options model data must be [JSON formatted](http://www.json.org).
-2. It must be accessible via HTTP(S) or on the local disk for the Rundeck server.
+1. The returned data must be [JSON formatted](http://www.json.org).
+2. It must be accessible via HTTP(S) or on the local disk for the Process Automation or Rundeck server.
 3. It must be in one of two JSON structures, _either_:
```

Comment:
"It must be in or contain a..." Since we can now support something below the root doc